src/extension/features/accounts/hide-memos/components/hide-memos-button.jsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import { controllerLookup } from 'toolkit/extension/utils/ember';
+import { l10n, getToolkitStorageKey } from 'toolkit/extension/utils/toolkit';
+
+export const HideMemosButton = ({ toggleHiddenState }) => {
+  const notHidden = getToolkitStorageKey('hide-memos', true);
+  const label = notHidden ? l10n('app.show', 'is-checked') : l10n('app.hide', 'not-checked');
```

Comment:
so `l10n` is for localization, it makes it so english strings can get translated to other languages.

The syntax for this should always be `l10n(<key>, <english string>)` meaning this would be

`l10n('app.show', 'Show')` and `l10n('app.hide', 'Hide)`, you'd want to use a different variable (`notHidden`) for the class name logic.

Reply:
is-checked is the name of the class YNAB has for those check boxes, that is where the switch is coming from.  should i just leave that as straight english then since it's for css not text?

Reply:
yeah made this comment lower down, I don't think these classes even do anything so you can just remove this logic